tv_apps/tv-ftu/style/index.css

```diff
@@ -0,0 +1,86 @@
+html {
+  font-size: 10px;
+  /* We have Fira Sans as default font in real device. But it isn't default in
+   * browser. That's the main reason this line is here.*/
+  font-family: "Fira Sans";
+  font-weight: 100;
+  font-style: italic;
+}
+
+html, body {
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  margin: 0;
+  padding: 0;
+  background: #000;
+}
+
+body {
+  background: url('images/background.png') no-repeat;
+}
+
+.txt-mid {
+  font-size: 3.8rem;
+}
+
+.txt-large {
+  font-size: 7rem;
+}
+
+.txt-center {
+  text-align: center;
+}
+
+smart-button[type=rounded-corner-text] {
+  width: 23.7rem;
+  font-size: 2.4rem;
+  font-weight: normal;
+}
+
+.slide {
+  width: 100%;
+  height: 100%;
+  color: #fff;
+  position: fixed;
+}
+
+.slide-section {
+  position: absolute;
+  margin-right: -50%;
+  left: 50%;
+  transform: translate(-50%, 0px);
+}
+
+.slide-section.top {
+  top: 7.5rem;
+}
+
+.slide-section.mid {
```

Comment:
(1) After discussion, focus style of the terms:
* Show the default scroll bar when scrolling the terms and no frame or outline effect when focusing on the terms
* Add one downward arrow icon to guide user scroll down to the terms and hide the arrow icon when scrolling down to the end of the terms

(2) Small screen issue:
Use SimpleKeyNavigation for button navigation to resolve the navigation issue